src/crosswalk/plots.py

```diff
@@ -196,8 +196,11 @@ def dose_response_curve(dose_variable, obs_method,
             marker=value[0], facecolors=value[1], edgecolors=value[2], 
             linewidth=0.6, alpha=.6, label=key
         )
-    plt.scatter(non_direct_df[f'{dose_variable}'], non_direct_df['y'],
-                facecolors='grey', edgecolors='grey', alpha=.3)
+
+    if len(non_direct_df):
```

Comment:
If we know `non_direct_df` is a data frame, let's use `if not non_direct_df.empty` :)